<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JxTaxMgrForm</class>
 <widget class="QDialog" name="JxTaxMgrForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>进项发票管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabHistory">
      <attribute name="title">
       <string>历史未认证发票</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,1">
       <item>
        <widget class="QTableWidget" name="twHistorys">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>所属凭证</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>开票日期</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>发票号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>税额</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>发票金额</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>客户</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="chkInited">
           <property name="text">
            <string>已初始化历史数据</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnInitHis">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>初始化历史数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveHis">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>保存历史数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddCurs">
           <property name="text">
            <string>添加本月未认证</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="rdbFilter">
           <property name="text">
            <string>查询发票号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtFilter">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="inputMask">
            <string>00000000</string>
           </property>
           <property name="maxLength">
            <number>8</number>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCurAuth">
      <attribute name="title">
       <string>本月认证发票</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,1">
       <item>
        <widget class="QTableWidget" name="twCurAuth">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>是否本月</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>发票号码</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>税额</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>发票金额</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>发票客户</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="btnSaveCur">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>保存</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCrtPz">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>创建调整凭证</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblInfos">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actDelHis">
   <property name="text">
    <string>删除记录</string>
   </property>
   <property name="toolTip">
    <string>删除历史记录</string>
   </property>
  </action>
  <action name="actDelCur">
   <property name="text">
    <string>移除记录</string>
   </property>
   <property name="toolTip">
    <string>移除本月认证记录</string>
   </property>
  </action>
  <action name="actJrby">
   <property name="text">
    <string>计入本月</string>
   </property>
   <property name="toolTip">
    <string>计入本月</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>JxTaxMgrForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>728</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
